rails/app/controllers/admin/themes_controller.rb

```diff
@@ -1,36 +1,5 @@
-module Admin  
+module Admin
   class ThemesController < Admin::ApplicationController
-    def create
-      if theme_params[:active] == "1"
-        deactivate_prev_theme
-      end
-      super
-    end
 
-    def update
-      if theme_params[:active] == "1"
-        deactivate_prev_theme
-      end
-      super
```

Comment:
context here? is there a way to preserve this work?

Reply:
A community only has one theme; most of this was so multiple themes could exist and get swapped which is no longer the case